packages/sage/src/commands/upgrade.ts

```diff
@@ -58,7 +58,8 @@ async function upgradeAction(options: UpgradeOptions) {
 	const update = await checkUpdates(packageJson, config, true)
 	logger.debug(`Update payload:`, update)
 
-	await updateRobo(plugins, config, options.autoAccept)
+	const autoAccept = options.yes
+	await updateRobo(plugins, config, autoAccept)
```

Comment:
Why not just pass `options.yes` inline instead of creating a dedicated variable used only once?

Reply:
Hey, just thought it would make things clearer, I can pass it as a one liner no issue if required